publiccode.yml

```diff
@@ -0,0 +1,61 @@
+# This repository adheres to the publiccode.yml standard by including this
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
+categories:
+  - it-development
+description:
+  it:
+    features:
+      - Generazione di publiccode.yml
+      - Modifica di publiccode.yml
+      - Validazione di publiccode.yml
+    genericName: Generatore di publiccode.yml
+    longDescription: |
+      This web app is meant to be used to generate a valid
+      [`publiccode.yml`](https://github.com/italia/publiccode.yml)  file. By
+      filling the form it automatically generates a YAML file compatible with
+      the latest version of the publiccode standard. Such a file may be copied
+      or downloaded locally in order to be inserted  in the root of the target
+      repository.
+
+
+      Furthermore, the app can be used as a validator. In fact, it is possible
+      to paste or import an already existing `publiccode.yml` file inside the
+      editor. The editor will validate the imported document and help fix any
+      existing issue.
+    shortDescription: An editor to generate and validate a publiccode.yml file
+developmentStatus: stable
+it:
+  conforme:
+    gdpr: false
+    lineeGuidaDesign: false
```

Comment:
Why false?

Reply:
Good point. I was not sure what that encompasses, is there an objective tool to evaluate conformance?

Reply:
@libremente repopened as #183 to try and trigger the GitHub action to no avail. I'll file an issue for that.